Add Ogaga's Builder Profile with Review Fixes

ogazboiz opened the pull request:
## Overview
This PR adds my builder profile page to the Batch 16 directory at `/builders/0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5`, addressing review feedback from #27. The page includes:

- A responsive profile with bio, contact info, and social links (Twitter, GitHub, Telegram)
- ETH address formatted using Scaffold-ETH’s `Address` component
- Fixes for reviewer comments to align with project standards

## Changes Addressing Review Feedback
- **New Branch**: Created `ogaga-profile-fixes` for this PR, per review request.
- **package-lock.json**: Removed, as the project doesn’t use npm lock files.
- **Image Optimization**: Replaced `<img>` with `next/image` to fix ESLint warning (`@next/next/no-img-element`) and optimize LCP.
- **SocialLinks**: Renamed to `ProfileSocialLinks` and moved to `packages/nextjs/app/builders/0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5/_components/` for page-specific context.
- **Page Typing/Naming**: Typed as `NextPage` and renamed to `OgagaProfile` for TypeScript clarity.

## Technical Details
- **Route**: `packages/nextjs/app/builders/0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5/page.tsx`
- **Component**: `ProfileSocialLinks` in page-specific `_components` folder
- **Standards**: Adheres to ESLint, Prettier, and TypeScript conventions
- **Dependencies**: No heavy assets or large dependencies added

## Preview
Preview the updated profile at:
- [Vercel Preview](https://batch16.buidlguidl.com/builders/0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5)

## Verification
- ✅ Renders correctly at all breakpoints (mobile, tablet, desktop)
- ✅ Links (Twitter, GitHub, Telegram, email) function properly
- ✅ Follows project conventions (linting, TypeScript, Scaffold-ETH)
- ✅ No `package-lock.json` included
- ✅ ESLint `<img>` warning resolved with `next/image`

## Notes
- Closes #27
- Thanks to @phipsae for the feedback! Please let me know if further changes are needed.
- Deployed to Vercel for preview (see link above).
- Awaiting maintainer approval for workflow merge.

Please review and approve. 🚀

phipsae commented on packages/nextjs/app/builders/0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5/page.tsx:
```diff
@@ -0,0 +1,348 @@
+// Ethereum address
+const myAddress = "0x2E15bB8aDF3438F66A6F786679B0bBBBF02A75d5";
+
+export default function BuilderPage() {
```
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.

Since there are more builder profiles, it might make sense to give your component a more specific name (e.g. "YourNameProfile")

phipsae reviewed (changes requested):
Thanks for new PR! 
Was there a specific reason to create a new PR? 
I closed the other one.

Left you comments below.

ogazboiz commented:
You said I should create a branch that is why

phipsae commented:
> You said I should create a branch that is why

True, thanks :)

ogazboiz commented:
Hi @phipsae ,  

Thanks for the feedback! 🙌 I’ve updated the component to use `NextPage` for better TypeScript integration and renamed it to `AkpoloOgagaProfile` for clarity.  

Here’s a preview of the UI for this profile page:  

### **UI Screenshot**  
# **dark mode ui**
![Scaffold-ETH 2 App](https://github.com/user-attachments/assets/7af13058-26df-47d7-b004-e920356d2f33)

# **lightmode ui**
![screenshot](https://github.com/user-attachments/assets/2a4f090d-f515-4466-b8e9-ae190279e240)


- Clean, responsive layout with sections for **About Me**, **Projects**, **Skills**, and **Contact**  
- Dynamic social links with custom icons  
- ETH address display using Scaffold-ETH’s `<Address>` component  
- Themed styling using CSS variables (e.g., `--color-primary`, `--color-base-100`)  

**Changes Made:**  
1. Typed the page as `NextPage`.  
2. renamed to AkpoloOgagaProfile

Let me know if you'd like any further adjustments!

phipsae commented:
Fixed an error in next.config.js to happend when you pulled in the lasted changes from main. But no worries :)

Lets merge
